modules/tensor_mechanics/include/utils/RankTwoScalarTools.h

```diff
@@ -27,6 +27,31 @@ MooseEnum invariantOptions();
 MooseEnum principalComponentOptions();
 MooseEnum mixedInvariantComponentOptions();
 
+enum class INVARIANT_TYPE
+{
+  VonMisesStress,
+  EffectiveStrain,
+  Hydrostatic,
+  L2norm,
+  VolumetricStrain,
+  FirstInvariant,
+  SecondInvariant,
+  ThirdInvariant,
+  TriaxialityStress,
+  MaxShear,
+  StressIntensity,
+  MaxPrincipal,
+  MidPrincipal,
+  MinPrincipal
+};
+
+enum class CYLINDRICAL_TYPE
```

Comment:
This is getting picky, but I think it would be a little more clear if this were renamed to `CYLINDRICAL_COMPONENT`